code/game/turfs/turf.dm

```diff
@@ -105,51 +104,24 @@
 	return 1 //Nothing found to block so return success!
 
 
-/turf/Entered(atom/atom as mob|obj)
-	..()
-//vvvvv Infared beam stuff vvvvv
-
-	if ((atom && atom.density && !( istype(atom, /obj/effect/beam) )))
-		for(var/obj/effect/beam/i_beam/I in src)
```

Comment:
It has to do with infrared beams---and no, it doesn't break them; they function fine still.